src/libraries/System.IO.Hashing/src/PACKAGE.md

```diff
@@ -2,43 +2,87 @@
 
 <!-- A description of the package and where one can find more documentation -->
 
+System.IO.Hashing offers a variety of hash code algorithms.
 
+Hash code algorithms are pivotal for generating unique values for objects based on their content, facilitating object comparisons, and detecting content alterations.
```

Comment:
Is it worth putting "these hash functions should not be used for security - e.g. passwords or verifying untrusted content" and linking to the System.Security.Cryptography much higher up?

Reply:
Great suggestion, will add that